### PR TITLE
doc: Describe ftp

### PR DESCRIPTION
**Pull request by holmanb**
## Proposed Commit Message
```
doc(nocloud): Describe ftp and ftp over tls implementation

Standardize and improve nocloud documentation.
```

### CODE REVIEW

File: doc/rtd/reference/datasources/nocloud.rst [hunk 1]
```diff
@@ -59,41 +76,45 @@ The permitted keys are:
...
-:ref:`user data formats<user_data_formats>` at the same base URL: ::
+The path pointed to by the URI will contain the following files:
 
-  /vendor-data
+``user-data`` (reqired)
+``meta-data`` (required)
+``vendor-data`` (optional)
 
+The ``user-data`` file uses :ref:`user data format<user_data_formats>`. The
+``meta-data`` file is a YAML-formatted file. The vendor data file adhers to
```
Comment (a-dubs) [inline doc/rtd/reference/datasources/nocloud.rst:116]:
```suggestion
``meta-data`` file is a YAML-formatted file. The vendor data file adheres to
```

**Comment by a-dubs** [comment]
changes look good! one typo fix to make tho.

File: doc/rtd/reference/datasources/nocloud.rst [hunk 2]
```diff
@@ -18,37 +18,54 @@ Configuration Methods:
...
-using kernel commandline arguments or SMBIOS serial number to tell cloud-init
-where on the filesystem to look.
+Method 2: Custom webserver
+--------------------------
 
-Alternatively, one can provide metadata via the kernel command line or SMBIOS
-"serial number" option. This argument might look like: ::
+Configuration files can be provided to cloud-init over HTTP(s). To tell
+cloud-init the URI to use, arguments must be passed to the instance via the
+kernel commandline  or SMBIOS serial number. This argument might look like: ::
```
Comment (TheRealFalcon) [inline doc/rtd/reference/datasources/nocloud.rst:33]:
```suggestion
kernel commandline or SMBIOS serial number. This argument might look like: ::
```

**Review by TheRealFalcon — APPROVED** [review]
I trust you can make mine and Alec's suggested changes. Nice cleanup too!

**Comment by holmanb** [comment]
Thanks for the reviews @a-dubs and @TheRealFalcon!

**Review by dermotbradley — COMMENTED** [review]
Doh, I forgot to submit my review.

**Pull request merged**